rules/windsurf.mdc

```diff
@@ -0,0 +1,9 @@
+---
+trigger: always_on
```

Comment:
So Windsurf needs this?

Ideally I'd think we'd use one rules file (that we can update in one place in gist or repo) and drop it at the location/filename needed for each tool, but perhaps that dream is not possible.